sdk/storage/Azure.Storage.Blobs/src/BlobServiceClient.cs

```diff
@@ -1667,6 +1665,156 @@ public BlobServiceClient(Uri serviceUri, TokenCredential credential, BlobClientO
         }
         #endregion UndeleteBlobContainer
 
+        #region RenameBlobContainer
+        /// <summary>
+        /// Renames an existing Blob Container.
+        /// </summary>
+        /// <param name="destinationContainerName">
+        /// The new name of the Blob Container.
+        /// </param>
+        /// <param name="sourceContainerName">
+        /// The name of the source container.
+        /// </param>
```

Comment:
nit: I'd expect description of these to be almost identical.

Reply:
fixed.